src/exampleTypes/CheckableResultTable.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 import _ from 'lodash/fp'
+import F from 'futil-js'
```

Comment:
This should be `futil` without `-js`. We publish under both package names because we didn’t have futil to start with, but we just recently moved this repo to the non -js version

Reply:
_very_ recently, actually - I think it was some last Friday, so definitely don't feel like you should have know about it 😂

I'll mention it at standup too